tests/test_self_packaging.py

```diff
@@ -18,17 +18,6 @@
     'PKG-INFO',
     'README.md',
     'pyproject.toml',
-    'setup.cfg',
-    'setup.py',
-    'src',
-    'src/build',
```

Comment:
flit doesn't create folder entries in the zip.

Reply:
Interesting, okay. Not complaining. :D